src/test/java/de/tum/in/www1/artemis/exam/ExamUtilService.java

```diff
@@ -470,24 +635,52 @@ public ExamSession addExamSessionToStudentExam(StudentExam studentExam, String s
         return examSession;
     }
 
-    public StudentExam addStudentExamForActiveExamWithUser(String user) {
+    /**
+     * Creates and saves a StudentExam with a registered User. Also creates and saves an empty course and an Exam that started 1 hour ago and ends in 1 hour.
```

Comment:
I think you shoud reformulate this to say that first a course and an exam (using the course and student users) are created and this exam is then used to create a student exam.

Reply:
reformulated 👍